_posts/2017-06-02-jslang-meetup-async-await.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title:  "Async Function Kata at #jslang Meetup"
+date:   2017-06-02 08:03:00 +0200
+categories: meetup javascript kata
+---
+
+The callback-hell times are long over. We have better tools now. But which of the new kids on the block do you want to 
+play with? In this meetup, last night we tried out [async functions][async-funcs], also known by the name "async/await". 
+It was a fun learning experience. Keep reading and get inspired.
+
+## The meetup
+["JavaScript The Language" (#jslang)][jslang-meetup] a hands-on meetup only about JavaScript the language. Not about any library, 
```

Comment:
> only about JavaScript the language. Not about any library

that sounds a bit off. I know what you meant but maybe you would want to write _"only about the JavaScript language as specified by ES5/ES6"_ or something similar otherwise the _the_ somehow seems misplaced after _JavaScript_ if you know what I mean ;--)